assembl/auth/social_auth.py

```diff
@@ -202,6 +202,12 @@ def get_setting(self, name):
     #     from social.pipeline.utils import partial_from_session
     #     return partial_from_session(self, session)
 
+    def build_absolute_uri(self, path=None):
+        path = super(AssemblStrategy, self).build_absolute_uri(path)
```

Comment:
I see that social_auth has this method, we have our own `get_route` and `frontendUrls` as well that tries to accomplish the same thing. Seems like a lot of ways to do the same thing. Understand that this method is bound to the PyramidStrategy, hence it's usage, but does the result differ from use of frontendUrls drastically?